scripts/docker-testnet/clean.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Delete the entire testnet folder, which includes configuration, executables and logs.
+
+export MULTIVERSXTESTNETSCRIPTSDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source "$MULTIVERSXTESTNETSCRIPTSDIR/variables.sh"
+
+echo "Stopping all containers..."
+docker stop $(docker ps -a -q)
+
+echo "Removing all containers..."
+docker container prune -f
```

Comment:
the scripts will most probably be used in development environment where there might be also other local containers running, i don't think it's fine to stop and prune all containers, we should stop only the intended containers